runHarness/ComputeResources/KubernetesCluster.pm

```diff
@@ -148,6 +148,19 @@ override 'getConfigFiles' => sub {
         print FILEOUT "$line\n";
     }
     close FILEOUT;
+				
+    $cmd = "kubectl get events --sort-by=.metadata.creationTimestamp";
```

Comment:
Shouldn't the kubeconfig and contextString also be provided in the command?
--kubeconfig=$kubeconfigFile $contextString
Also, should probably get the events for all the namespaces to enable more info:
 --all-namespaces

Reply:
Nice catch